bordertale/Screens.cs

```diff
@@ -56,37 +54,40 @@ public static void TitleScreen()
             while (inLoop)
             {
                 string option = PrintUtils.Input().ToLower();
-                switch (option)
```

Comment:
This should be left as a switch case, an if else with that many cases takes too long to trace.

Reply:
I've replaced the series of if-then statements I made with the original switch. The wording of the issue had me believe all switches had to be changed. 

Also, the project seems nice. I like the RPG genre, although I've only ever tried one or two text command style ones.